assets/js/modules/analytics-4/datastore/audiences.test.js

```diff
@@ -2407,7 +2395,7 @@ describe( 'modules/analytics-4 audiences', () => {
 			it( 'should return `undefined` if the available audiences are not loaded', () => {
 				registry
 					.dispatch( MODULES_ANALYTICS_4 )
-					.receiveGetSettings( {} );
+					.startResolution( 'getAvailableAudiences', [] );
```

Comment:
We can just change this to use `receiveGetAudienceSettings()`. Please check the other cases where `startResolution()` has been applied, this might have been due to the check for `undefined` in `*getAvailableAudiences()` and it seems likely they can be changed to use `receiveGetAudienceSettings()` which is a bit simpler to understand.

```suggestion
					.receiveGetAudienceSettings( {} );
```